### DISCUSSION TITLE
173

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#137] [General]
# 潮流周刊第173期 - 布面油画

封面图拍摄来源于曾梵志的一幅布面油画，细看越看越好看，这幅画比较特殊，创作的时候，把对应展览的墙和地上都扩展到了，故意为之的艺术感。

https://weekly.tw93.fun/posts/173-%E5%B8%83%E9%9D%A2%E6%B2%B9%E7%94%BB

**Comment by HeroadZ** [comment]
感谢分享，请问您用的翻译pdf的软件是什么啊。

**Reply by Runtu4378** [reply]
浏览器扩展：沉浸式翻译

**Reply by HeroadZ** [reply]
感谢！原来是我的idm自动下载导致无法在chrome中加载，所以一直没发现沉浸式翻译还有翻译pdf的功能。
